src/llmcompressor/arg_parser/recipe_arguments.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+@dataclass
+class RecipeArguments:
```

Comment:
is the user-configured dataclass that gets "compiled" into something like `GPTQModifier` or `QuantizationModifier`? I'm wondering if these concepts can be merged?

Reply:
Can you elaborate, I dont understand

Reply:
Discussed with @horheynm that is refactored from having `recipe` and `recipe_args` as inputs elsewhere, and not a net-new abstraction